RussianFilter/sections/specific.txt

```diff
@@ -5904,7 +5904,7 @@ www.ex.ua/*/index.css
 ||civic-club.ru/static/partners/
 ||clock.ru/company/$domain=watch.ru
 ||cloudfront.net^$domain=2baksa.net|imgsrc.ru|nowa.cc
-||cloudfront.net^$domain=allfon.org|allfon.pro|allfon.xyz|batpic.com|kinovideo.tv|mintmanga.com|mobzon.ru
+||cloudfront.net^$domain=allfon.org|allfon.pro|allfon.xyz|batpic.com|kinovideo.tv|mintmanga.com|mobzon.ru|qaru.site
```

Comment:
too wide rule.  cloudfront.net is not ad server. Better block `||d1uw69x4c2zrim.cloudfront.net^`